templates/dalite_settings.py

```diff
@@ -0,0 +1,92 @@
+import getpass
+
+import os
+
+SECRET_KEY = '{{ DALITE_SECRET_KEY }}'
+
+DEBUG = False
+
+ALLOWED_HOSTS = ['*']
```

Comment:
My omission, mostly. Bear in mind that (since 30 minutes ago ;) nginx also filters requests to drop improper domains.